Description: Disable Wayland temporarily as dde-wayland is not ready.
Author: Arun Kumar Pariyar <openarungeek@gmail.com>
Last-Update: 2020-11-03
---

Index: deepin-qt5dxcb-plugin/qt5platform-plugins.pro
===================================================================
--- deepin-qt5dxcb-plugin.orig/qt5platform-plugins.pro
+++ deepin-qt5dxcb-plugin/qt5platform-plugins.pro
@@ -1,4 +1,3 @@
 TEMPLATE = subdirs
 SUBDIRS += \
     xcb/xcb.pro \
-    wayland/wayland.pro
